### PR TITLE
Spacing

### LINKED ISSUE #1153: Triple dot button too small
**FastHub Version: 4.4.2**  
**Android Version: 7.0 (SDK: 24)**  
**Device Information:**  
- **Manufacturer:** LGE  
- **Brand:** lge  
- **Model:** LG G6  
---

I need to have ninja precision to hit the triple dot button. Otherwise I will hit the scroll bar or even smiley button. 

image

_Sent from my LG G6 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

### PR DESCRIPTION
**Pull request by casz**
fixes #1153 
Really, just a matter of spacing

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-rc1'
+        classpath 'com.android.tools.build:gradle:3.0.0-rc2'
```
Comment (yakov116) [inline build.gradle]: I don't know if you did, but always check with @k0shk0sh before updating the gradle files.

**Comment by yakov116** [comment]
@casz Will try it out now.

Thanks!

File: build.gradle (hunk 1 above)
Comment (casz) [reply build.gradle]: I can always take it out 👍

Comment (yakov116) [reply build.gradle]: No worries you can leave it since I will wait for him to merge. Just so you know in the future. 
I learned the hard way 🌵

**Comment by casz** [comment]
Added some more spacing to file review too 👍

**Comment by casz** [comment]
At least with these changes I cannot accidentally engage the scrollbar, so from me 👍

**Comment by casz** [comment]
Cheers!

**Pull request merged**